packages/flutter/lib/src/material/page.dart

```diff
@@ -12,31 +12,38 @@ const double _kMinFlingVelocity = 1.0;  // screen width per second
 
 // Used for Android and Fuchsia.
 class _MountainViewPageTransition extends AnimatedWidget {
-  static final FractionalOffsetTween _kTween = new FractionalOffsetTween(
-    begin: FractionalOffset.bottomLeft,
-    end: FractionalOffset.topLeft
-  );
-
   _MountainViewPageTransition({
     Key key,
-    Animation<double> animation,
+    this.routeAnimation,
     this.child,
   }) : super(
     key: key,
     listenable: _kTween.animate(new CurvedAnimation(
-      parent: animation, // The route's linear 0.0 - 1.0 animation.
+      parent: routeAnimation, // The route's linear 0.0 - 1.0 animation.
       curve: Curves.fastOutSlowIn
     )
   ));
 
+  static final FractionalOffsetTween _kTween = new FractionalOffsetTween(
+    begin: FractionalOffset.bottomLeft / 4.0,
```

Comment:
it's not a big deal, but i think this will probably get evaluated each time. you could instead do `const FractionalOffset(0.0, 0.25)` or some such.